src/api/app/helpers/webui/announcement_helper.rb

```diff
@@ -0,0 +1,19 @@
+module Webui::AnnouncementHelper
+  def announcement_content(announcement)
```

Comment:
Please don't. You can easily do this in the view.

Reply:
That's a leftover from when we had 2 places to render the ToS. We can remove it now